src/lib/services/ifc-wasm-parser.ts

```
@@ -1,135 +1,152 @@
+                });
+                await pyodide.loadPackage(["micropip", "numpy"]);
+                const micropip = pyodide.pyimport("micropip");
+                await micropip.install(
+                    "https://cdn.jsdelivr.net/gh/IfcOpenShell/wasm-wheels@main/ifcopenshell-0.8.2+d50e806-cp312-cp312-emscripten_3_1_58_wasm32.whl"
+                );
+                resolve(pyodide);
+            }
+        } catch (err) {
+            reject(err);
+        }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)